tests/integration/remote/invoke/test_remote_invoke.py

```diff
@@ -131,26 +202,38 @@ def setUpClass(cls):
         cls.stack_name = f"{TestMultipleResourcesInvoke.__name__}-{uuid.uuid4().hex}"
         cls.create_resources_and_boto_clients()
 
-    def test_invoke_empty_event_provided(self):
-        command_list = self.get_command_list(stack_name=self.stack_name, resource_id="EchoEventFunction")
+    @parameterized.expand([("EchoEventFunction", {})])
+    def test_invoke_empty_event_provided(self, resource_id, expected_response):
+        command_list = self.get_command_list(stack_name=self.stack_name, resource_id=resource_id)
 
         remote_invoke_result = run_command(command_list)
         self.assertEqual(0, remote_invoke_result.process.returncode)
         remote_invoke_result_stdout = json.loads(remote_invoke_result.stdout.strip().decode())
-        self.assertEqual(remote_invoke_result_stdout, {})
+        self.assertEqual(remote_invoke_result_stdout, expected_response)
 
     @parameterized.expand(
         [
-            ("HelloWorldServerlessFunction", {"message": "Hello world"}),
-            ("EchoCustomEnvVarFunction", "MyOtherVar"),
-            ("EchoEventFunction", {"key1": "Hello", "key2": "serverless", "key3": "world"}),
+            (
+                "HelloWorldServerlessFunction",
+                '{"key1": "Hello", "key2": "serverless", "key3": "world"}',
+                {"message": "Hello world"},
+            ),
+            ("EchoCustomEnvVarFunction", '{"key1": "Hello", "key2": "serverless", "key3": "world"}', "MyOtherVar"),
+            (
+                "EchoEventFunction",
+                '{"key1": "Hello", "key2": "serverless", "key3": "world"}',
+                {"key1": "Hello", "key2": "serverless", "key3": "world"},
+            ),
+            # Skip remote invoke Step function integration tests
```

Comment:
is this not needed or will be un-commented in the future?

Reply:
This will be un-commented once step functions is supported for `remote invoke`.  Trying to think of a way to not have to comment them and skip them based on the resource-type.